<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>initialWindow</class>
 <widget class="QMainWindow" name="initialWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(244, 249, 250);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="initialwindow_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(
    x1: 0, y1: 0, x2: 1, y2: 1,
    stop: 0 rgba(196, 236, 236, 1), stop: 1 rgba(255, 255, 255, 1)
);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>201</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/icon.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Arial&quot;;
color: white; 
background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(35, 120, 99, 1), stop: 1 rgba(77, 206, 175, 1)
);

border-radius: 10%;</string>
     </property>
     <property name="text">
      <string>Start Recording</string>
     </property>
    </widget>
    <widget class="QPushButton" name="aboutButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>360</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border: none;
color: black;</string>
     </property>
     <property name="text">
      <string>About PerfectPitch</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modeButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>20</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:none;
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/lightMode.png</normaloff>../images/lightMode.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>300</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Arial&quot;;
color: white; 
background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(35, 120, 99, 1), stop: 1 rgba(77, 206, 175, 1)
);

border-radius: 10%;</string>
     </property>
     <property name="text">
      <string>Start Listening</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
